src/scenes/Projects/Create/CreateProject.tsx

```diff
@@ -0,0 +1,31 @@
+import { useSnackbar } from 'notistack';
+import React from 'react';
+import { Except } from 'type-fest';
+import { ButtonLink } from '../../../components/Routing';
+import { useCreateProjectMutation } from './CreateProject.generated';
+import {
+  CreateProjectForm,
+  CreateProjectFormProps as Props,
+} from './CreateProjectForm';
+
+export const CreateProject = (props: Except<Props, 'onSubmit'>) => {
+  const [createProject] = useCreateProjectMutation();
```

Comment:
Here's where we should invalidate the project list cache. @richardgirges thoughts?